lib/elastomer/version_support.rb

```diff
@@ -151,9 +165,18 @@ def unsupported_indexing_directives
       @unsupported_indexing_directives = unsupported_keys.each_with_object({}) do |key, h|
         h[key] = "_#{key}"
       end
+      check_op_type(@unsupported_indexing_directives) # hack: valid param loses underscore between ES 2.x & 5.x
       @unsupported_indexing_directives.freeze
     end
 
+    # COMPATIBILITY
+    # If _op_type param is found in map of symbols to request param names, check for ES 5.x and shim
+    def check_op_type(params = {})
+      if es_version_5_x? && params.key?(:op_type)
+        params[:op_type] = "op_type"
```

Comment:
Isn't this going to overwrite whatever was there with `"op_type"`?

Reply:
yeah in the case of the `check_op_type` soon to be `fix_op_type`, this is to maintain the accepted/bad param lists across the versions we're building there. those symbol lists all get mapped to dictionaries of `:key` = `"_key"` for convenience but this one breaks the mold :(

Reply:
Oh right! I remember that `:key => "_key"` business. Carry on :)